api/inc/box_config.h

```diff
@@ -59,14 +59,14 @@ UVISOR_EXTERN void const * const main_cfg_ptr;
         acl_list_count \
     }; \
     \
-    UVISOR_EXTERN const __attribute__((section(".keep.uvisor.cfgtbl_ptr_first"), aligned(4))) void * const main_cfg_ptr = &main_cfg;
+    UVISOR_EXTERN const __attribute__((section(".keep.uvisor.cfgtbl_ptr_first"), aligned(4))) void * const public_box_cfg_ptr = &public_box_cfg;
 
 /* Creates a global page heap with at least `minimum_number_of_pages` each of size `page_size` in bytes.
  * The total page heap size is at least `minimum_number_of_pages * page_size`. */
 #define UVISOR_SET_PAGE_HEAP(page_size, minimum_number_of_pages) \
     const uint32_t __uvisor_page_size = (page_size); \
     uint8_t __attribute__((section(".keep.uvisor.page_heap"))) \
-        main_page_heap_reserved[ (page_size) * (minimum_number_of_pages) ]
+        public_box_page_heap_reserved[ (page_size) * (minimum_number_of_pages) ]
```

Comment:
Consider renaming to `public_page_heap_reserved`, for consistency with calling this the `public page heap`.